src/vs/platform/update/electron-main/updateService.snap.ts

```diff
@@ -160,28 +160,20 @@ export class SnapUpdateService extends AbstractUpdateService2 {
 	protected doCheckForUpdates(context: any): void {
 		this.setState(State.CheckingForUpdates(context));
 
+		type UpdateNotAvailableClassifcation = {
```

Comment:
this type is declared many times

Reply:
Yes it's a small type and I didn't know of a good common place to put it.

Reply:
https://github.com/Microsoft/vscode/blob/fe3e0cdcdeb4a7f5bc2745bda73564ba4ddc75b7/src/vs/platform/update/electron-main/abstractUpdateService.ts

Reply:
or https://github.com/Microsoft/vscode/blob/653280d133bf36f182189ded3506d3603d09fcd7/src/vs/platform/update/common/update.ts